File: c_backup/BIN/C Prog/sumofdigits.c
// determining the sum of digits
#include<stdio.h>
int main()
{
	int i=243,j,sum=0;
	while(i!=0)
	{
		j=i%10;
		i=i/10;
		printf("%d",j);
		//sum=sum+j;
		//if(i!=0)
		//printf(" + ");		
	}
	//printf(" = %d",sum);
	return 0;
}

File: java_backup/java.jimut/p_prog/finalpracexam/sumofdigits.java
package finalpracexam;


class sumofdigits
{
public static void main(int n)
{
int a,s=0;
while(n>0)
{
a=n%10;
s=s+a;
n=n/10;
}
System.out.println("THE SUM OF DIGITS="+s);
}
}